packages/create-snowpack-app/index.js

```diff
@@ -151,9 +157,25 @@ if (requiredVersion < 10) {
     cwd: targetDirectory,
     stdio: "inherit",
   };
-  const npmInstallProcess = useYarn
-    ? execa("yarn", ["--silent"], npmInstallOptions)
-    : execa("npm", ["install", "--loglevel", "error"], npmInstallOptions);
+
+  function installProcess(packageManager) {
+    switch (packageManager) {
+      case "npm":
+        return execa(
+          "npm",
+          ["install", "--loglevel", "error"],
+          npmInstallOptions
+        );
+      case "yarn":
+        return execa("yarn", ["--silent"], npmInstallOptions);
+      case "pnpm":
+        return execa("pnpm", ["install"], npmInstallOptions);
+      default:
+        break;
```

Comment:
this will fail on the next line when we try to access `npmInstallProcess.stdout`. Feel free to add back that `throw new Error()` line you had earlier

Reply:
OK, I am not very familiar with Node.js, I added back `return new Error("unspecified installer");` from the previous commit, I hope that was what you were referring to.